internal/configuration/validator/authentication.go

```diff
@@ -16,6 +16,46 @@ func validateFileAuthenticationBackend(configuration *schema.FileAuthenticationB
 	if configuration.Path == "" {
 		validator.Push(errors.New("Please provide a `path` for the users database in `authentication_backend`"))
 	}
+
+	if configuration.Algorithm == "" {
+		configuration.Algorithm = schema.DefaultFileAuthenticationBackendConfiguration.Algorithm
+	} else {
+		configuration.Algorithm = strings.ToLower(configuration.Algorithm)
+		if configuration.Algorithm != "argon2id" && configuration.Algorithm != "sha512" {
+			validator.Push(fmt.Errorf("Unknown hashing algorithm supplied, valid values are argon2id and sha512, you provided '%s'", configuration.Algorithm))
```

Comment:
Let's use consistent language, we've got:
- you provided
- you specified
- you set

I'm happy with either configured or specified.

```suggestion
			validator.Push(fmt.Errorf("Unknown hashing algorithm supplied, valid values are argon2id and sha512, you configured '%s'", configuration.Algorithm))
```

Reply:
Done like this for all instances. Chosen Configured.